.circleci/run_e2e.sh

```diff
@@ -3,18 +3,21 @@
 CNI_HUB=${CNI_HUB:-istio}
 CNI_TAG=${CNI_TAG:-$1}
 
+chartdir=/go/helm
+
 cd /go/src/istio.io/istio
 
+# Install istio-cni prior to executing the Istio e2e test.  Now that the helm chart for istio/istio no longer
+# depends on the istio-cni chart, we need to explicitly do these steps.
+helm init --client-only
+#helm repo add istio.io https://storage.googleapis.com/istio-release/releases/1.1.0-snapshot.6/charts
```

Comment:
I think you should remove the comments here.   Thinking about this more the tests here should be only about using the istio source code not also validating if the the CNI charts have been properly pushed to the istio charts repo.  The tests on the istio side can confirm that.   In this repo it is probably cleaner if it tests the CNI code AND the CNI charts at the tip of the branch but pull in the Istio code to confirm that.

Reply:
Is this comment just to remove lines 13 & 14 or to remove all the other comments above?  Lines 10 & 11 are still valid comments.  I could reword to explain that we're testing with the istio-cni chart from the PR under-test.